example-projects/default-config/doc/spec.md

```diff
@@ -0,0 +1,6 @@
+# Tracing Example
+`dsn~example~1`
+
+Example requirement
+
+Needs: utest, impl
```

Comment:
These are separate, different test projects. I want to test the plugin, not OFT.